Competition2017/src/competition/subsystems/autonomous/DriveToHopperThenBoilerCommandGroup.java

```diff
@@ -0,0 +1,81 @@
+package competition.subsystems.autonomous;
+
+import com.google.inject.Provider;
+import competition.subsystems.drive.commands.DriveForDistanceCommand;
+import competition.subsystems.drive.commands.DriveToOrientationCommand;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import xbot.common.command.BaseCommand;
+import xbot.common.properties.BooleanProperty;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class DriveToHopperThenBoilerCommandGroup extends CommandGroup{
+    public DoubleProperty distanceFromWallToBaseline;
+    public DoubleProperty distanceFromTurningPointToHopper;
+    public DoubleProperty distanceFromTurningPointToBoiler;
+    public DoubleProperty headingToFaceHopper;
+    public DoubleProperty waitTimeForFuelCollection;
+    public DoubleProperty headingToFaceBoiler;
+    public BooleanProperty goToBoiler;
+    
+    private class TimeoutCommand extends BaseCommand {
```

Comment:
You should probably just add this command to the SeriouslyCommonLib, with a public setTimeout method.